demo/repo/Jenkinsfile

```diff
@@ -17,26 +17,31 @@ parallel(longerTests: {
     runTests(servers, 20)
 })
 
-stage name: 'Staging', concurrency: 1
-node {
-    servers.deploy 'staging'
-}
+milestone ordinal: 1
```

Comment:
Ups, sorry. No, it does not. As both parameters (`label` and `ordinal`) are optional, then the name has to be explicitly set.